packages/core/src/shared/logger/toolkitLogger.ts

```diff
@@ -120,6 +123,9 @@ export class ToolkitLogger extends BaseLogger implements vscode.Disposable {
     }
 
     override sendToLog(level: LogLevel, message: string | Error, ...meta: any[]): number {
+        // XXX: force debug/verbose logs for Beta users.
+        level = this.isBeta ? 'info' : level
```

Comment:
I'm not familiar with this code so I can't tell if i'm missing something, but isn't this saying if its beta then use info? I thought we wanted verbose/debug if its beta users

Reply:
It forces the "sinks" (log providers such as Output channel, file, etc.) to log at "info" level. This forces the vscode Output channel to show logs that it would otherwise hide.